src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[]
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+}
+
+export const Tabs: FC<Props> = (props) => {
+  const { tabs, selectedTabId, onTabSelected } = props;
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                key={id}
+                className={cn({
+                  'is-active': id === selectedTab.id,
```

Comment:
You have comparisons like `id === selectedTab.id` twice here, so you just could create a corresponding value like `const isSelected = id === selectedTab.id`

Reply:
Good point! Actually it's pretty similar check

```
const isSelected = id === selectedTab.id;

isSelected // id === selectedTab.id
!isSelected // id !== selectedTab.id
```

Reply:
My fault. Fixed)